templates/login.html

```diff
@@ -61,20 +61,27 @@
     </a>
     {% else %}
     <a role="button" class='btn btn-jupyter btn-lg' href='{{authenticator_login_url}}'>
-      Log in to continue
+      Log in para continuar
     </a>
     {% endif %}
   </div>
 
   <div class="col-md-8 col-md-offset-2 details">
     <div class="col-md-6">
       <big class="details-welcome lead">
-      Welcome to the <a href="{{ custom.org.url }}">{{ custom.org.name }}</a> <strong>2i2c JupyterHub</strong>.
+        Bienvenido al <strong>JupyterHub-2i2c</strong> de <a href="{{ custom.org.url }}">{{ custom.org.name }}</a>
+        del evento Intercoonecta OceanHackWeek en español, Feb. 27 - Mar. 3, 2023.
+        <br />
+        (<strong>The interface will be in Spanish by default through March 3, 2023! If you are not 
+        participating in the OHW-español event Feb. 27 - Mar. 3, please use this hub lightly.</strong>)
```

Comment:
How about a link to directions on how to switch language? We could include them as a subheading on https://oceanhackweek.org/resources/prep/jupyterhub.html or more generically send folks to https://jupyterlab.readthedocs.io/en/stable/user/language.html for now.

Reply:
> I think this works, though I'd love to have an English overview of the event that we could link folks too, along with instructions on how to change the JupyterLab language.

We don't have the web site for the event ready to go yet. I'll work on an initial version over the next couple of days, and we can include a short-paragraph description in English.

> Logo

I do plan to submit a changed logo that incorporates the modified OHW logo we've been using.

> Multi-lingual instructions/links to localize interface (since we also have a notable Brazilian contingent)

I think I'm now pedalling back from changing the language default, and instead stick with the English default plus instructions for switching the language. We already have the instructions in Spanish for switching the language to Spanish. Anyways, the details are off topic here, but I'll tweak the relevant text in this PR.

Reply:
@abkfenris How does my new text look?

Once you and I come to agreement about these initial changes to the login page, I'll request that the PR be merged, per @consideRatio's instructions.